src/app/about-us/members/members.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-members',
+  templateUrl: './members.component.html',
+  styleUrls: ['./members.component.scss']
+})
+export class MembersComponent implements OnInit {
+  members = [
+    {
+      name: 'Claudia Martinez',
+      mainCareer: 'about-us.members.career.informatics',
+      currentOcupation: 'about-us.members.ocupation.dev',
```

Comment:
maybe you can change this only to be labels:

```javascript
{
  ...
  currentOcupation: 'dev'
  ...
}
```

and in the HTML do: 
```html
<p>{{ 'about-us.members.ocupation.' + member.currentOcupation | translate }}</p>
```
I'm recommending this as this data is going to be saved in firebase and i'm thinking it won't be useful in the future that our data reflect full paths of translations. Let me know what do you think?